src/screens/WalletScreen.js

```diff
@@ -86,6 +93,20 @@ export default function WalletScreen() {
     briefSectionsData: walletBriefSectionsData,
   } = useWalletSectionsData();
 
+  useEffect(() => {
+    // This is the fix for Android wallet creation problem.
+    // We need to remove the welcome screen from the stack.
+    if (ios) {
+      return;
+    }
+    const isWelcomeScreen =
```

Comment:
inb4, no need for the optional chaining, the navigation structure is semi-static here